pytest_plugins/metadata_markers.py

```diff
@@ -165,7 +165,7 @@ def pytest_collection_modifyitems(items, config):
     logger.info('Processing test items to add testimony token markers')
     for item in items:
         item.user_properties.append(
-            ("start_time", datetime.datetime.utcnow().strftime(FMT_XUNIT_TIME))
+            ("start_time", datetime.datetime.now(datetime.UTC).strftime(FMT_XUNIT_TIME))
```

Comment:
Given that https://docs.python.org/3/library/datetime.html#datetime.UTC was added in Python 3.11, the 3.10 check is failing. The following patch would need to be made to support Python 3.10.
```suggestion
            ("start_time", datetime.datetime.now(datetime.timezone.utc).strftime(FMT_XUNIT_TIME))
```